site/content/docs/5.1/examples/footers/footers.css

```diff
@@ -1,11 +1,7 @@
-/* Boosted mod */
-.example-footer-custom-form .btn:active {
-  border-color: var(--bs-orange) !important; /* stylelint-disable-line declaration-no-important */
+.b-example-divider {
```

Comment:
`.b-example-divider` is already defined in `site/layouts/_default/examples.html` so I believe you can remove this file and remove the following lines in the Markdown file:
```md
extra_css:
  - "footers.css"
```